plasmapy/analysis/nullpoint.py

```diff
@@ -957,31 +982,29 @@ def endpoint_sign_check(curve_endpoints, curve_name):
             index = 1
         elif curve_name == "z":
             index = 2
-        if (
-            np.sign(
-                tlApprox(
-                    curve_endpoints[0][0], curve_endpoints[0][1], curve_endpoints[0][2]
-                )[index]
-            )
-            * np.sign(
-                tlApprox(
-                    curve_endpoints[1][0], curve_endpoints[1][1], curve_endpoints[1][2]
-                )[index]
-            )
-            > 0
+
+        first_endpoint = tlApprox(
+            curve_endpoints[0][0], curve_endpoints[0][1], curve_endpoints[0][2]
+        )[index]
+        second_endpoint = tlApprox(
+            curve_endpoints[1][0], curve_endpoints[1][1], curve_endpoints[1][2]
+        )[index]
+        if np.isclose(first_endpoint, 0, atol=_EQUALITY_ATOL) or np.isclose(
+            second_endpoint, 0, atol=_EQUALITY_ATOL
         ):
+            return True
+        if np.sign(first_endpoint) * np.sign(second_endpoint) > 0:
             return False
         else:
             return True
```

Comment:
```suggestion
        return False if np.sign(first_endpoint) * np.sign(second_endpoint) > 0 else True
```
A possible alternative.  If you think the original way is more readable, feel free to skip this suggestion.